lib/marginalia/comment.rb

```diff
@@ -159,7 +159,7 @@ def self.database
         end
 
         def self.connection_config
-          return if marginalia_adapter.pool.nil?
+          return unless marginalia_adapter.pool.respond_to?(:spec)
```

Comment:
Does this work on older Rails as well?

Reply:
Sorry for the delay. I haven't tried this on older rails myself, but as mentioned this should retain backwards compatiblity — `nil` also doesn't respond to `spec`. If you merge #90 first then I can rebase on there to prove that it works. :-)

Reply:
(The test suite does pass on my local machine when run against all available gemfiles.)